app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -0,0 +1,53 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+import { parseCondition } from '@data-driven-forms/react-form-renderer';
+
+const createSchema = (fields, edit, loadSchema) => {
```

Comment:
Consume the passed `!!storageManagerId` here:
```suggestion
const createSchema = (fields, edit, ems, loadSchema) => {
```